### PR TITLE
feat: container-registry vulnerabilities

### PR DESCRIPTION
**Pull request by glimberea**
## What does this fix or implement?

Added vulnerability feature to `container-registry` and new `artifacts` and `vulnerabilities` resources.

## Checklist

- [ ] PR name added as appropriate (e.g. `feat:`/`fix:`/`doc:`/`test:`/`refactor:`)
- [ ] Tests added or updated
- [ ] Documentation updated
- [ ] Sonar Cloud Scan
- [ ] Changelog updated and version incremented (label: upcoming release)
- [ ] Github Issue linked if any
- [ ] Jira task updated

### CODE REVIEW

**Comment by avirtopeanu-ionos** [comment]
could you please also update the changelog?

File: commands/container-registry/repository/list.go [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+	_ = c.Command.RegisterFlagCompletionFunc(
+		constants.FlagRegistryId,
+		func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			return registry.RegsIds(), cobra.ShellCompDirectiveNoFileComp
+		},
+	)
+
+	c.AddSetFlag(
+		cloudapiv6.ArgOrderBy, "", "-lastPush", []string{
+			"-lastPush", "-lastPull", "-artifactCount", "-pullCount", "-pushCount", "name", "lastPush",
```
Comment (avirtopeanu-ionos) [inline commands/container-registry/repository/list.go:56]: Are these minuses intended? `-lastPull` and `lastPull`? does this order them in reverse order?

File: commands/container-registry/repository/repository.go [hunk 2]
```diff
@@ -18,46 +31,52 @@ func RegRepoDeleteCmd() *core.Command {
...
-func PreCmdDelete(c *core.PreCommandConfig) error {
-	err := core.CheckRequiredFlags(c.Command, c.NS, "name", "registry-id")
+func RepositoryNames(registryId string) []string {
+	repos, _, err := client.Must().RegistryClient.RepositoriesApi.RegistriesRepositoriesGet(
+		context.Background(),
+		registryId,
+	).Execute()
 	if err != nil {
-		return err
+		return nil
```
Comment (avirtopeanu-ionos) [inline commands/container-registry/repository/repository.go:74]: is this change intended?

**Review by avirtopeanu-ionos — APPROVED** [review]

File: commands/container-registry/repository/repository.go (hunk 2 above)
Comment (glimberea) [reply commands/container-registry/repository/repository.go:74]: yes, the `return err` is from a function that was originally in this file and that was moved to another file (repository/delete.go)

File: commands/container-registry/repository/list.go (hunk 1 above)
Comment (glimberea) [reply commands/container-registry/repository/list.go:56]: Yes, the minuses are intended and they do exactly what you assumed. When used without minus, the results are in ascending order and with the minus, they are in descending order.
